packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -682,6 +683,54 @@ describe('TransactionController', () => {
       expect(controller.state.transactions[0].status).toBe(
         TransactionStatus.unapproved,
       );
+      expect(
```

Comment:
Should we avoid duplicating assertions so we can limit the failed tests if a "feature" breaks in future?

Reply:
This should be addressed in latest commit